src/cloudStorageProvider.js

```diff
@@ -145,6 +156,15 @@ class CloudStorageProvider {
                 vscode.window.showErrorMessage(`Failed to create a file: ${error.message}`);
             }
         } else {
+            // remove reference line before saving
+            const config = vscode.workspace.getConfiguration('playcanvas');
+
+            if (config.get('usePlaycanvasTypes') && asset.file.filename.endsWith('.js')) {
+                let strContent = new TextDecoder().decode(content);
+                strContent = strContent.substring(this.typesReference.length);
```

Comment:
should we compare what we're removing is exatly what we expected to remove?

Reply:
ah it was accidentally removed, restored now